templates/liquid/Measure.liquid

```diff
@@ -127,7 +127,7 @@
         {% if Measure.author.exists() %}
         <tr>
             <th scope="row"><b>Author: </b></th>
-            <td style="padding-left: 4px;">{{Measure.author}}</td>
+            <td style="padding-left: 4px;">{{Measure.author.name}}</td>
         </tr>
         {% endif %}
 
```

Comment:
This looks good for Measure.author, but we should treat all of the ContactDetail-valued elements the same way then, update the templates for Measure.editor, Measure.reviewer, and Measure.endorser to follow suit

Reply:
Also note that these same elements appear in the other templates (Library, PlanDefinition, and ActivityDefinition) we should update those to be consistent as well